src/server.tsx

```diff
@@ -79,9 +91,7 @@ const startServer = async () => {
   server.app.use(router.allowedMethods());
   server.app.use(async (ctx: any, next: any) => {
     if (ctx.request.path === '/robots.txt') {
-      if (ctx.request.hostname.toLowerCase() === 'freeboardgame.org') {
-        ctx.response.body = OPEN_ROBOTS_TXT;
-      } else {
+      if (ctx.request.hostname.toLowerCase() !== 'freeboardgame.org') {
         ctx.response.body = RESTRICTIVE_ROBOTS_TXT;
```

Comment:
We can probably remove the RESTRICTIVE and OPEN_ROBOTS_TXT constant too

Reply:
OPEN_ROBOTS_TXT is not present.

RESTRICTIVE_ROBOTS_TXT is served when the Host header is not freeboardgame.org, which I thought was desired behavior.